framework/ldtest/test_logger.go

```diff
@@ -10,12 +10,13 @@ import (
 	"github.com/fatih/color"
 )
 
-var consoleTestErrorColor = color.New(color.FgYellow)              //nolint:gochecknoglobals
-var consoleTestFailedColor = color.New(color.FgRed)                //nolint:gochecknoglobals
-var consoleTestFailedNonCriticalColor = color.New(color.FgYellow)  //nolint:gochecknoglobals
-var consoleTestSkippedColor = color.New(color.Faint, color.FgBlue) //nolint:gochecknoglobals
-var consoleDebugOutputColor = color.New(color.Faint)               //nolint:gochecknoglobals
-var allTestsPassedColor = color.New(color.FgGreen)                 //nolint:gochecknoglobals
+var consoleTestErrorColor = color.New(color.FgYellow)                   //nolint:gochecknoglobals
+var consoleTestFailedColor = color.New(color.FgRed)                     //nolint:gochecknoglobals
+var consoleTestFailedNonCriticalColor = color.New(color.FgYellow)       //nolint:gochecknoglobals
+var consoleTestSkippedColor = color.New(color.Faint, color.FgBlue)      //nolint:gochecknoglobals
+var consoleFailedDebugOutputColor = color.New(color.Faint, color.FgRed) //nolint:gochecknoglobals
+var consolePassedDebugOutputColor = color.New(color.Faint)              //nolint:gochecknoglobals
```

Comment:
Previously, the debug output for tests was always gray. Now, it'll be reddish-gray/pink for failed tests, gray for passed tests (you only see the debug output for passed tests if you specify `-debug-all`, but it's pretty verbose so now the failures will stand out more).